presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -15,6 +15,11 @@ The driver is also available from Maven Central:
         <version>\ |version|\ </version>
     </dependency>
 
+.. note::
+  If you’re familiar with the different implementations of JDBC drivers,
+  the Presto JDBC driver is a Type 4 driver. This means it talks to the
+  Presto native protocol.
```

Comment:
Is this relavant or just geeky? :)

Reply:
I think this can be left off..

Reply:
I don't think this is useful. The important part about being a type 4 driver is that it's pure Java. However, for the past 10-20 years every use type 4, so I don't think we need to add it.

Reply:
OK, this note is out (despite its lofty origin in The Definitive Guide).

Reply:
hahah..